tests/pipeline/test_blaze.py

```diff
@@ -614,73 +623,57 @@ def test_cols_with_some_missing_vals(self):
-            "str_value": np.array(["a",
-                                   None,
-                                   "c",
-                                   "a",
-                                   None,
-                                   "c",
-                                   "a",
-                                   "b",
-                                   "c"],
-                                  dtype='object'),
-            "float_value": np.array([0,
-                                     np.NaN,
-                                     2,
-                                     0,
-                                     np.NaN,
-                                     2,
-                                     0,
-                                     1,
-                                     2],
-                                    dtype='float64'),
-            "int_value": np.array([1,
-                                   0,
-                                   3,
-                                   1,
-                                   0,
-                                   0,
-                                   1,
-                                   2,
-                                   0],
-                                  dtype='int64'),
-            "bool_value": np.array([True,
-                                    False,
-                                    True,
-                                    True,
-                                    False,
-                                    False,
-                                    True,
-                                    True,
-                                    False],
-                                   dtype='bool'),
-            "dt_value": [pd.Timestamp('2011-01-01'),
-                         pd.NaT,
-                         pd.Timestamp('2011-01-03'),
-                         pd.Timestamp('2011-01-01'),
-                         pd.NaT,
-                         pd.Timestamp('2011-01-03'),
-                         pd.Timestamp('2011-01-01'),
-                         pd.Timestamp('2011-01-02'),
-                         pd.Timestamp('2011-01-03')],
-        },
-            columns=['str_value', 'float_value', 'int_value', 'bool_value',
-                     'dt_value'],
-            index=pd.MultiIndex.from_product(
-                (self.dates, self.asset_finder.retrieve_all(
-                    self.ASSET_FINDER_EQUITY_SIDS
-                ))
-            )
+        expected = pd.DataFrame(
+            {
+                'str_value': np.array(
+                    ['a', None, 'c', 'a', None, 'c', 'a', 'b', 'c'],
+                    dtype='object',
+                ),
+                'float_value': np.array(
+                    [0, np.NaN, 2, 0, np.NaN, 2, 0, 1, 2],
+                    dtype='float64',
+                ),
+                'int_value': np.array(
+                    [1, 0, 3, 1, 0, 3, 1, 2, 3],
+                    # [1, 0, 3, 1, 0, 0, 1, 2, 0],
```

Comment:
these were the changes I had questions about, we talked in person and we discussed the new ffill algorithm to preserve the old behavior